### ISSUE TITLE
Color contrast ratio of "Remote" text is 4.381:1 which is less than 4.5:1: A11y_Azure Function_Default screen_Color Contrast

### ISSUE THREAD

**Issue by Rama8345** [microsoft/vscode-azurefunctions#4332]
### GIT Hub Tags:
#A11yTCS; #A11ySev2; #Azure Tools for VS Code; #BM_Azure Tools for VS Code_Win32_Nov2024; #DesktopApp; #A11yMAS; #WCAG1.4.3; #ColorContrast; #Win11; #FTP;

### Environment and OS details:
Application name: Azure Tools for VS Code
URL: Visual studio code

### Reproduction Steps:

1. Open Visual studio code
2. TAB till extension button and press ENTER then search for "Azure Functions"
3. TAB till install and press ENTER.
4. TAB till "Azure" button and press ENTER.
5. TAB till "Resource" section.
6. Open accessibility insights for windows and check color contrast ratio of "Remote" text.
7. Observe the issue whether Color contrast ratio of "Remote" text is 4.381:1 which is less than 4.5:1 or not.

### Actual behavior:
Color contrast ratio of "Remote" text is 4.381:1 which is less than 4.5:1

### Expected behavior:
Color contrast ratio of "Remote" text should be greater than or equals to 4.5:1

### User Impact:
Low vision users will not be able to see "Remote" text if color contrast ratio will be less than 4.5:1.

### Attachment:
![Image](https://github.com/user-attachments/assets/15d04a9a-0fee-444a-9388-8b3266592ac1)

**Comment by PoojaNamde** [comment]
#Rev:PoojaNamde;

**Comment by MicroFish91** [comment]
Could you let me know which VS Code dark theme you are using here?

F1 > Preferences: Color Theme

**Comment by PoojaNamde** [comment]
@MicroFish91 Default dark theme we used to log this bug.

**Comment by MicroFish91** [comment]
Hi, could you double verify the contrast ratio on this.  I just ran a quick test and got > 4.5.

![Image](https://github.com/user-attachments/assets/0315370a-f09e-4833-b38e-eca080c0af17)

**Comment by AnKushSingh05** [comment]
Verified the bug in below version
Version: 1.97.0-insider (user setup)
Issue is not fixed.
![Image](https://github.com/user-attachments/assets/530b2544-f155-49ba-aaa2-4dfcd2b32248)

**Comment by MicroFish91** [comment]
Okay thank you, tracking upstream issue posted on VS Code repo https://github.com/microsoft/vscode/issues/237130

**Comment by PoojaNamde** [comment]
@MicroFish91 We have logged new bug https://github.com/microsoft/vscode-azurefunctions/issues/4481 to track this issue. Could you please close this bug.
